static/js/metrics-explorer.js

```diff
@@ -27,8 +27,13 @@ let availableMetrics = [];
 let previousStartEpoch = null;
 let previousEndEpoch = null;
 let rawTimeSeriesData=[];
-let allFunctions=[];
+let allFunctions,functionsArray =[];
+var availableOptions = ["max by", "min by", "avg by", "sum by", "count by", "stddev by", "stdvar by", "group by"];
```

Comment:
nit: we should name this var a little differnetly , it is availableOptions for what ?